# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-xcb-surface.c
# Commit: 972a07ec6164
# Full Hash: 972a07ec616459f697d835e0ed870ecbf9d822b5
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 1
# Description:
#   Bug 1892913 - patch 8 - Update and apply 04-subpixel-aa-api.patch r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209415
# ==============================================================================

diff -r dcf06dfaff0c -r 972a07ec6164 gfx/cairo/cairo/src/cairo-xcb-surface.c
--- a/gfx/cairo/cairo/src/cairo-xcb-surface.c	Mon May 06 19:25:16 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-xcb-surface.c	Mon May 06 19:25:16 2024 +0000
@@ -906,7 +906,8 @@
 				       cairo_scaled_font_t          *scaled_font,
 				       cairo_glyph_t                *glyphs,
 				       int                           num_glyphs,
-				       cairo_bool_t                  overlap)
+				       cairo_bool_t                  overlap,
+				       cairo_bool_t                  permit_subpixel_antialiasing)
 {
     cairo_xcb_surface_t *surface = (cairo_xcb_surface_t *) extents->surface;
     cairo_surface_t *fallback = _cairo_xcb_surface_fallback (surface, extents);